frontend/services/unsupportedBrowserDetection.ts

```diff
@@ -3,15 +3,15 @@ import UAParser from 'ua-parser-js';
 
 const DEFAULT_SUPPORTED_BROWSERS_CONFIG = {
 	desktop: [{
-		browser: 'firefox', minversion: 41,
+		browser: 'firefox', minversion: 54,
```

Comment:
Opera 38+ is missing from the list of supported browsers.

![Screenshot 2020-09-10 at 11 23 40](https://user-images.githubusercontent.com/1438332/92717597-61b2c980-f358-11ea-888d-d26b6e62ff8a.png)

Reply:
@Charence Thanks for pointing this. Opera with the supported version was added to the list.